fenix/app/src/main/java/org/mozilla/fenix/settings/search/SearchEngineFragment.kt

```diff
@@ -109,7 +115,6 @@ class SearchEngineFragment : PreferenceFragmentCompat() {
                     getString(R.string.app_name),
                 )
             }
-
```

Comment:
nit let's not remove this new line

Reply:
eek will do, sorry, lemme fix up the spacing and extra lines. Tyty for taking the time to review 🙂